src/test/java/ch/uzh/ifi/hase/soprafs23/controller/UserControllerTest.java

```diff
@@ -264,6 +271,13 @@ public void postUsersUserIdLogout_unauthorized() throws Exception {
 
     @Test
     public void testGetUserByIdReturns200() throws Exception {
+        User user = new User();
+        user.setId(1L);
+        user.setUsername("firstUsername");
+        user.setPassword("firstPassword");
+        user.setToken("firstToken");
+        user.setStatus(UserStatus.ONLINE);
+        user.setAllergiesSet(new HashSet<>(Arrays.asList("garlic", "nuts")));
```

Comment:
aren't these lines in the @BeforeEach? I think only the last one setting the allergies is necessary for this test, and maybe putting that one in the @BeforeEach as well would be better

Reply:
jep, leftover from writing the tests new. changed/shorten it